test/pytest/service-desktop/test_threads.py

```diff
@@ -1,45 +1,130 @@
-# Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+# Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
```

Comment:
consider renaming to message threads from threads, title might be misleading - as one can think i.e. about software threads

Reply:
:+1:

Reply:
Renamed